src/porepy/numerics/mixed_dim/assembler.py

```diff
@@ -120,12 +120,20 @@ def assemble_matrix_rhs(
             (g1, data_1, data_edge, local_matrix)
 
         """
+        # Define the matrix format, common for all the sub-matrices
+        if matrix_format == "csr":
+            sps_matrix = sps.csr_matrix
+        elif matrix_format == "csc":
+            sps_matrix = sps.csc_matrix
+        else: # coo
+            sps_matrix = sps.coo_matrix
```

Comment:
Are you sure coo is a good default value?

Reply:
not really, should we put csr?